app/scripts/modules/core/src/cluster/cluster.service.ts

```diff
@@ -176,6 +176,10 @@ export class ClusterService {
     });
   }
 
+  public isDeployingArtifact(cluster: ICluster): boolean {
+    return (cluster.imageSource === 'artifact');
```

Comment:
The parens are valid JS here but not really idiomatic (you'll only see it if you're using React).